test/TripRepo-test.js

```diff
@@ -42,10 +42,52 @@ describe('TripRepo', () => {
 
   describe('getFolioByYear', () => {
 
-    it('should return a new TripRepo of all the trips that match a given userID', function() {
+    it('should return a new TripRepo of all the trips that match a given userID', () => {
 
       expect(repo.getFolioByYear(2019)).to.deep.equal(new TripRepo(data, []));
       expect(repo.getFolioByYear(0)).to.deep.equal(new TripRepo([], []));
     });
   });
+
+  describe.only('getPastFolio', () => {
+
+    it('should return a new TripRepo of all the trips that have ended already', () => {
+
+      let date = new Date(2019, 8, 25);
+      expect(repo.getPastFolio(date)).to.deep.equal(repo);
+
+      let bad = new Date(2019, 8, 24);
+      expect(repo.getPastFolio(bad)).to.deep.equal(new TripRepo([], []));
+    });
+  });
+
+  describe.only('getCurrentFolio', () => {
+
+    it('should return a new TripRepo of all the trips that have ended already', () => {
+
+      let beg = new Date(2019, 8, 16);
+      let middle = new Date(2019, 8, 20);
+      let end = new Date(2019, 8, 24);
+      expect(repo.getCurrentFolio(beg)).to.deep.equal(repo);
+      expect(repo.getCurrentFolio(middle)).to.deep.equal(repo);
+      expect(repo.getCurrentFolio(end)).to.deep.equal(repo);
+
+      let before = new Date(2019, 8, 15);
+      let after = new Date(2019, 8, 25);
+      expect(repo.getCurrentFolio(before)).to.deep.equal(new TripRepo([], []));
+      expect(repo.getCurrentFolio(after)).to.deep.equal(new TripRepo([], []));
+    });
+  });
+
+  describe.only('getUpcomingFolio', () => {
+
+    it('should return a new TripRepo of all the trips that have ended already', () => {
```

Comment:
One big thing I see is that the test names for 54, 66,84 are all the same but are doing different things.